tests/wdio/spec/rsi-saverestore.spec.js

```diff
@@ -0,0 +1,251 @@
+      expect(questionnaireElementExists).to.equal(true)
+      const submitBtn = browser.element('.qa-btn-submit')
+      submitBtn.waitForExist(10000)
+      browser.setValue('#fad63234-1083-4f6a-826a-20b5df6e4baa-year', '2016')
+      browser.setValue('#c8c4bd92-fd45-4fd1-83b6-18c813de2df2-year', '2017')
+      const saveandcontinueBtn2 = browser.element('.qa-btn-submit')
+      saveandcontinueBtn2.click();
+      submitBtn.waitForExist(10000)
+
+      browser.setValue('.input--currency', 'Test')
+
+      const saveandcontinueBtn3 = browser.element('.qa-btn-submit')
+      saveandcontinueBtn3.click();
+
+      const erroralert2 = browser.element('.alert__body')
+      const Q2text = erroralert2.getText()
+
+      const experror2 = 'Please only enter whole numbers into the field.'
+
+      if(Q2text.indexOf(experror2) > -1) {
+        console.log('Expected Text is present: ' + experror2 );
+      }else {
+        console.log(Q2text.indexOf(experror2) !== -1);
+        assert.equal(Q2text, experror2);
+      }
+
+  //    browser.debug()
+      browser.call(done)
+    })
+    // Revisiting 0102
+    it('Progress from the developer page for 1_0102', function() {
+      const userId = '.qa-user-id'
+      const collectionSID = '.qa-collection-sid'
+      const selectSchema = '.qa-select-schema'
+      const periodID = 'period_id'
+      const periodString = '.period_str'
+      browser.url('/dev')
+      browser.waitForExist(userId)
+      browser.setValue(userId, 'yoganandkunche')
+      browser.waitForExist(collectionSID)
+      browser.setValue(collectionSID, '7890102')
+      browser.waitForExist(selectSchema)
+      browser.selectByValue(selectSchema, '1_0102.json')
+      browser.click('.qa-btn-submit-dev')
+    })
+
+    it('To make sure the error page has been reached for 0102 revisit', function(done) {
+      const saveandcontinueBtn4 = browser.element('.qa-btn-submit')
+      saveandcontinueBtn4.waitForExist(10000)
+      const erroralert = browser.element('.alert__body')
+      const Q1text = erroralert.getText()
+      const experror = 'The \'to\' date must be different to the \'from\' date.'
+      if(Q1text.indexOf(experror) > -1) {
+        console.log('Expected Text is present: ' + experror );
+      }else {
+        console.log(Q1text.indexOf(experror) !== -1);
+        assert.equal(Q1text, experror);
+      }
+      browser.call(done)
+    })
+
+    it('Correcting the error and submitting the 0102 form', function(done) {
+      const questionnaireElementExists = browser.isExisting('.qa-questionnaire-form')
+      expect(questionnaireElementExists).to.equal(true)
+      const submitBtn = browser.element('.qa-btn-submit')
+      submitBtn.waitForExist(10000)
+      browser.setValue('[id = "94f368e4-7c6c-4272-a780-8c46328626a2-year"]', '2016')
+      browser.setValue('#dc156715-3d48-4af3-afed-7a0a6bb65583-year', '2017')
+
+      const saveandcontinueBtn2 = browser.element('.qa-btn-submit')
+      saveandcontinueBtn2.click();
```

Comment:
You can probably just do this `browser.element('.qa-btn-submit').click()` instead of assigning to a `const` all the time. You only really need to do that when you want to keep a reference around to do multiple things to the button.